info/templates/info/resources.html

```diff
@@ -1,5 +1,9 @@
 {% extends "base.html" %}
-
+{% load static %}
+{% block stylesheets %}
+    <link rel="stylesheet" href="{% static 'info/css/sidebar.css' %}">
+{% endblock %}
```

Comment:
Why is the sidebar being added to a template that isn't part of the dashboard? The sidebar is used to navigate the dashboard.